CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
 - PR #1319 CSV Reader: Use column wrapper for gdf_column output alloc/dealloc
 - PR #1376 Change series quantile default to linear
 - PR #1399 Replace CFFI bindings for NVTX functions with Cython bindings
+- PR #1386 Added macros `GDF_TRY()`, `CUDF_TRY()` and `ASSERT_CUDF_SUCCEEDED()`
```

Comment:
The line for #1386 shouldn't be in this PR.

Reply:
@harrism : Not exactly; my initial comment mentions a dependency on the yet-unmerged code. Let me now drop that.

Reply:
So, now the PR only has the code that comes on top of PR #1386. Which means that it will only build after #1386 is merged and I rebase this PR on branch-0.7.

Reply:
@eyalroz it's fine to have this PR depend on another PR by merging it in.